Mathlib/Data/Option/Basic.lean

```diff
@@ -477,4 +477,8 @@ lemma isNone_eq_false_iff (a : Option α) : Option.isNone a = false ↔ Option.i
 lemma eq_none_or_eq_some (a : Option α) : a = none ∨ ∃ x, a = some x :=
   Option.exists.mp exists_eq'
 
+lemma eq_none_iff_forall {o : Option α} : (∀ (x : α), some x ≠ o) ↔ o = none := by
```

Comment:
```suggestion
lemma forall_ne_iff_eq_none {o : Option α} : (∀ (x : α), some x ≠ o) ↔ o = none := by
```
(or perhaps `forall_some_ne_iff_eq_none`). Similar in the other new `ne` lemmas.

Thanks!

bors d+

Reply:
Ops, you're right. Sorry. I'll do it later.